core/src/test/resources/log4j.properties

```diff
@@ -14,5 +14,8 @@
 # limitations under the License.
 ##
 
-log4j.rootLogger=OFF
+log4j.rootLogger=INFO, stdout
```

Comment:
I suggest we leave it. I agree that noisy tests are difficult to parse, but these tests aren't too noisy and I find the output to be helpful when debugging.